README.md

```diff
@@ -1 +1,146 @@
-# automatization
\ No newline at end of file
+# n8n and Temporal Docker Compose Setup
+
+This repository contains a Docker Compose configuration to run n8n and Temporal services together.
+
+## Services
+
+The setup includes:
+
+- **n8n**: An automation tool that allows you to create workflows visually
+- **Temporal**: A workflow orchestration platform with the following components:
+  - Temporal server
+  - Temporal UI
+  - PostgreSQL (database)
+  - Elasticsearch (for visibility features)
```

Comment:
The documentation mentions `Elasticsearch`, but the docker-compose.yml actually uses OpenSearch